packages/api-queries/src/plugins.ts

```diff
@@ -0,0 +1,8 @@
+import { fetchUserSitesPlugins } from '@automattic/api-core';
+import { queryOptions } from '@tanstack/react-query';
+
+export const pluginsQuery = () =>
+	queryOptions( {
+		queryKey: [ 'me-sites-plugins' ],
```

Comment:
It would be better to split it into individual items so we can clear the cache by the prefix like `'me'`

```js
queryKey: [ 'me', 'sites-plugins' ],
```